lib/_javascripts/markdownClassAppender.js

```diff
@@ -0,0 +1,37 @@
+const markdown = require('markdown-it');
+
+function decorate(md, ruleName, cssClass) {
+
+    // Remember the old renderer if overridden, or proxy to the default renderer.
+    let defaultRender = md.renderer.rules[ruleName] || function (tokens, idx, options, env, self) {
+        return self.renderToken(tokens, idx, options);
+    };
+  
+    md.renderer.rules[ruleName] = function (tokens, idx, options, env, self) {
+        // Add a new `target` attribute, or replace the value of the existing one.
+        tokens[idx].attrJoin('class', cssClass);
+
+        // console.log(`Adding CSS class "${cssClass}" for ${ruleName}`);
+    
+        // Pass the token to the default renderer.
+        return defaultRender(tokens, idx, options, env, self);
+    };
+}
+
+module.exports = (md, tokens=[], appendRules=true, cssClass='md') => {
+    //use all tokens unless specified
+    let ruleNames = Array.from(tokens);
+    if (appendRules) {
```

Comment:
Could split this out into its own function that takes tokens, md and appendRules